Almost there:
```
Run yarn --pure-lockfile
yarn install v1.22.10
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error @tagoro9/eslint-config-typescript@1.0.0: The engine "node" is incompatible with this module. Expected version "^12.14.1". Got "14.16.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.3.4",
-    "@tagoro9/eslint-config-typescript": "1.0.0",
+    "@tagoro9/eslint-config-typescript": "^1.1.0",
     "@types/conventional-commits-parser": "^3.0.0",
     "@types/git-url-parse": "^9.0.0",
     "@types/jest": "^25.1.4",

